Docs/source/Hydrodynamics.rst

```diff
@@ -297,7 +297,11 @@ Source Terms
 We now compute explicit source terms for each variable in :math:`\Qb` and
 :math:`\Ub`. The primitive variable source terms will be used to construct
 time-centered fluxes. The conserved variable source will be used to
-advance the solution. We neglect reaction source terms since they are
+advance the solution. To reduce memory usage, we do not include source terms
```

Comment:
I would suggest putting this new text in a `.. note::` so it stands out

also add an index entry, 

`.. index:: USE_SPECIES_SOURCES`